client/src/app/pages/applications/components/bulk-copy-assessment-review-form/bulk-copy-assessment-review-form.tsx

```diff
@@ -103,11 +103,9 @@ export const BulkCopyAssessmentReviewForm: React.FC<
       type: FilterType.multiselect,
       selectOptions: dedupeFunction(
         applications
+          .filter((app) => !!app.businessService?.name)
           .map((app) => app.businessService?.name)
-          .map((name) => ({
-            key: `BulkCopyAssessmentReviewForm-${name}`,
```

Comment:
I verified I was still able to copy an assessment, so I'm not certain what impact this was supposed to have.